addons/survey/views/survey_templates.xml

```diff
@@ -212,6 +212,11 @@
                         <t t-if="not survey_last">Continue</t>
                         <t t-else="">Submit</t>
                     </button>
+                    <button t-if="skipped_questions" type="submit" class="btn btn-primary"
+                            t-att-value="'finish' if len(skipped_questions) == 1 else 'next_skipped'">
+                        <t t-if="len(skipped_questions) == 1">Submit</t>
+                        <t t-else="">Next Skipped</t>
```

Comment:
We need to either not display anything in the else case, or hide the original submit button. Currently you get 2 submit buttons in the survey